docs/_documentations/troubleshooting.md

```diff
@@ -504,7 +504,25 @@ Issue link: https://github.com/eclipse/codewind-docs/issues/64
 ## Node.js and Swift templates remain in the starting state
 The templates `Appsody Node.js template` and `Appsody Swift template` remain in the starting state by default because these templates do not have a server implemented, and therefore, its status cannot be detected. These templates do not have a server and are intended to help you implement your own server.
 
-**Workaround** To get the application into a **Started** state, use a server for the application. After the application has a server, Codewind can monitor the server, and the status turns to **Started** if the server is running. Alternatively, you can also temporarily [stop Codewind from continueously pinging the application](#how-to-stop-the-app-from-continuously-pinging).
+**Workaround** To get the application into a **Started** state, use a server for the application. After the application has a server, Codewind can monitor the server, and the status turns to **Started** if the server is running. Alternatively, you can also temporarily [stop Codewind from continuously pinging the application](#how-to-stop-the-app-from-continuously-pinging).
+
+<!--
+Action/Topic: Appsody with Codewind
+Issue type: bug/info
+Issue link:
+-->
+## Kafka templates remain in the starting state
+
+The `Kafka` templates for the Appsody `Quarkus` and `Spring Boot` application stacks remain in the starting state when you create a project using them. These projects require users to configure a Kafka instance to connect to in order to run correctly. Additionally, these templates do not expose an endpoint at the `/` path that Codewind attempts to ping, since these projects are not meant to be REST-style applications.
```

Comment:
When you say `...when you create a project using them.` It took me a moment to determine that you referred to `Kafka templates` instead of `Appsody Quarkus` and `Spring Boot`. So, to ensure users will comprehend correctly, I recommend saying, `...when you create a project using Kafka templates.`

Reply:
In the last two sentences here, we say, `These projects require users to configure...` then we say, `these templates do not expose an endpoint...` 

Are we talking about the Kafka templates or the projects created with the Kafka templates? If we talk about templates, use `templates` for the subject of both sentences. If we talk about the projects, use `projects` as the subject of both sentences.

Reply:
when we say, `....Codewind attempts to ping, since these projects...` I recommend we say `...Codewind attempts to ping because these projects are not meant to be REST-style applications.` I think it conveys the causal relationship better.

Reply:
Re 1st comment, the sentence already begins with "The Kafka templates..".  If we also end with "... using Kafka templates", does it sound repetitive or redundant?

Reply:
Good point and nice catch. Let's leave it as is then!